packages/atlas/src/themesource/atlas_core/web/core/widgets/_scroll-container.scss

```diff
@@ -0,0 +1,83 @@
+//
+// DISCLAIMER:
+// Do not change this file because it is core styling.
+// Customizing core files will make updating Atlas much more difficult in the future.
+// To customize any core styling, copy the part you want to customize to styles/web/sass/app/ so the core styling is overwritten.
+//
+
+@mixin scroll-container() {
+    /* ==========================================================================
+       Scroll Container
+
+       Default Mendix Scroll Container Widget.
+    ========================================================================== */
+    .mx-scrollcontainer-wrapper:not(.mx-scrollcontainer-nested) {
+        -webkit-overflow-scrolling: touch;
+    }
+
+    .mx-scrollcontainer-horizontal {
+        width: 100%;
+        display: table;
+        table-layout: fixed;
+    }
+    .mx-scrollcontainer-horizontal > div {
+        display: table-cell;
+        vertical-align: top;
+    }
+
+    .mx-scrollcontainer-nested {
+        padding: 0;
+    }
+    .mx-scrollcontainer-fixed > .mx-scrollcontainer-middle > .mx-scrollcontainer-wrapper,
```

Comment:
is this generated?
is `.mx-scrollcontainer-fixed .mx-scrollcontainer-wrapper` selector not sufficient if it is applied for all?

Reply:
No, `.mx-scrollcontainer-fixed .mx-scrollcontainer-wrapper` is going to change the behavior. We have `top` and `bottom` regions as well.